plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -768,6 +768,7 @@ protected function save_object( $request, $creating = false ) {
 			if ( $creating ) {
 				$object->set_created_via( 'rest-api' );
 				$object->set_prices_include_tax( 'yes' === get_option( 'woocommerce_prices_include_tax' ) );
+				$object->save();
```

Comment:
Wouldn't this trigger hooks that are not meant to be triggered? Where is the source of the error?